src/main/java/org/opensearch/security/identity/labels/DefaultUserInfoLabelingRule.java

```diff
@@ -0,0 +1,59 @@
+package org.opensearch.security.identity.labels;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.opensearch.action.search.SearchRequest;
+import org.opensearch.common.util.concurrent.ThreadContext;
+import org.opensearch.core.common.transport.TransportAddress;
+import org.opensearch.search.labels.rules.Rule;
+import org.opensearch.security.dlic.rest.support.Utils;
+import org.opensearch.security.user.User;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Rules to get user info labels for RuleBasedLabelingService in OpenSearch
+ */
+public class DefaultUserInfoLabelingRule implements Rule {
+    public static final String REMOTE_ADDRESS = "remote_address";
+    public static final String USER_NAME = "user_name";
+    public static final String USER_SECURITY_ROLES = "user_backend_roles";
+    public static final String USER_ROLES = "user_roles";
+    public static final String USER_TENANT = "user_tenant";
```

Comment:
Are all these fields needed? From the linked issue, it seems like only tenant information is needed.

Reply:
Hey @DarshitChanpura ! The `tenancy` we want to obtain is different from the current tenancy concept (a dashboard concept ?). We want to provide the ability for users to identify "who send what" queries. A request can be sent from a human user (identified by username, roles etc), it can also be sent by some application used by multiple users (identified by username, remote address etc), Thus we want to populate all information that can correctly identify a "source of a request".